packages/AlgernonTests.package/ALGTestClassReload.class/instance/setUp.st

```diff
@@ -0,0 +1,6 @@
+running
+setUp
+
+	super setUp.
+	className := #ALGDummyClass.
+	self deleteClass.
```

Comment:
```suggestion
	self deleteClass
```

Reply:
@Olliwehr Fyi, you could just have clicked the "commit suggestion" button here, then you will not have to have created a new commit (Futur II in English is hard😅)

Reply:
@bewee i did exactly that and just edited the commit message. ^^ But as far as i noticed that will always produce a new commit, no?

Reply:
@Olliwehr Ahh ok, I'm sorry then. You confused me by knowing that one can change the commit message there😅😂
Always nice to learn new stuff :D